src/vmm/src/builder.rs

```diff
@@ -169,8 +170,14 @@ impl Display for StartMicrovmError {
     }
 }
 
-// Wrapper over io::Stdin that implements `ReadableFd`.
+// Wrapper over io::Stdin that implements `Serial::ReadableFd` and `vmm::CustomData`.
 struct SerialStdin(io::Stdin);
+impl SerialStdin {
+    /// Returns a `SerialStdin` wrapper over `io::stdin`.
+    pub fn get() -> Self {
```

Comment:
`new` seems a more appropriate name here.

Reply:
Idiomatic naming implies that `new` returns a **new** instance of an object.

In this scenario we're in a bit of a gray area:
technically this function does provide a **new** wrapper over the Stdin handle, but the wrapper is used as the stdin-handle itself which is a globally unique object, also the wrapper has no state other than the global handle.

I chose `SerialStdin::get()` to hint at the fact that this is a unique global object and not an independent instantiation. I propose we keep it as `get` or find another name better suited than `new`.

Reply:
I find the code to read nicer as it was before (i.e. without the `get` function). Is this function really needed?